drivers/net/ppp.c

```diff
@@ -206,6 +365,18 @@ static int ppp_send_flush(struct ppp_driver_context *ppp, int off)
 	 */
 	if (IS_ENABLED(CONFIG_GSM_MUX)) {
 		(void)uart_fifo_fill(ppp->dev, buf, off);
+	} else if (IS_ENABLED(CONFIG_NET_PPP_ASYNC_UART)) {
```

Comment:
This seems to be incompatible with muxing, could you add suitable lines to Kconfig so that this is enforced (user cannot select both at the same time)

Reply:
Good point. Added.